deploy/f2f-spec.yaml

```diff
@@ -0,0 +1,864 @@
+          description: >-
+            the OAuth client identifier which has been registered with CI CRI
+    WriteTxmaRequest:
+      type: object
+      required:
+        - event_name
+      properties:
+        event_name:
+          type: string
+          description: The name of the txma event being written to SQS
+          example: F2F_AUTH_CODE_ISSUED
+          enum:
+            - F2F_SESSION_CREATED
+            - F2F_SESSION_ABORTED
+            - F2F_AUTH_CODE_ISSUED
+            - F2F_ACCESS_TOKEN_ISSUED
+  parameters:
+    SessionHeader:
+      name: session-id
+      in: header
+      description: A UUID generated by the Session API to act as a primary key for the Session in DynamoDB
+      required: true
+      schema:
+        type: string
+    ResponseType:
+      in: query
+      name: response_type
+      example: code
+      required: true
+      description: the type of authorization response that is expected
+      schema:
+        type: string
+    ClientId:
+      in: query
+      name: client_id
+      example: xxx
+      required: true
+      description: the OAuth client identifier which has been registered with CI CRI
+      schema:
+        type: string
+    Request:
+      in: query
+      name: request
+      example: eyJhbGciOiJSU0EtT0FFUC0yNTYiLCJlbmMiOiJBMjU2R0NNIn0.AISIEvimTkm_1gfG9gQA5btB7lHu2riU90KI-UyuH35CABCJk_KKUTsptasmNG5n6Zo6jL8vlMrj3y4A44nDrnTeH_OGLByerCd-EKNx9EZrplcsQ8VM8dip7LCUQy10OGH6ggoMGq9TbiC7lP_8md8UgvcNzd_zm6q6Ade5zd_sUzTWNSX3q6lVfpkpuMdj-2l18zk-yJEvaDArP7U1VBT8531ixxW1lUKflTHlyC-t6B-45R04o3KubhVb-_gVt7KKXo7UQUY-wmHiTUGsxWb3xrROKs6tv4Ug7Oi1MVngesL8CahjyLamvk-tdS1ji2rxn47Xs2Q21FQf3cHPtQ.9bNdWPzw2Cf6uyR_.VVj96Ejcc84yKv3gfHY6Nond9FJrQ-AoQ7pQSlmnzicoJ7hpHwyHUuNiSJNO4q631VKAblNbdprAA11PcYL3f_hcocxwORR8e3tQlDG9TFn6lxg3l9M34ccJrkD51ippIpsChQhIFqS2hU5W9BFyOPdpQhlhtc7R-d0zqbylh-z-snzxRG7Q-hVsBQpOzOLMDWcJNOCHlF8kZjp4dlWYFj3Yl_9HAa-hrY-DpoxdkvXVIrBdJarD77Bwwxo8y1IWEgCuDA8t5ZmrZMY9fxoufQg9jW5D7hB3qysgSo3S-6SFyt3rP_WD5ecezRrTCY-voDeWBtfdwuKo5UUaPidN4ut1n8SXAdoANxgbuEhyABjvMESi3AnWwZRx8yh3xEMVSwZ8kFQ4cVeBW2hrkiSXMjL1Wtz6nhhY9itFcmzBU4JMvoYfdf-ZZP-F4ph_Xh9CRS7rBzmrgUt81LahzYfCY91lS93qT7r3Wrh0OfIb0vvIkVUebDdCZw25PoG2oREty4k59z7j9ho8VqbtSS6_kpJGWx8Sikut1I-bVduk79Vf1MwEGxSRvZ_pyZHxxZDe19GPfamhctFXMcebyJoT2zwWhcW4AWFSUtm2JI36sZhFcsdRIRBfSjJ4MNXdlrHdOyhvo8SwTWpfO5lU9yyF8lANbXxjdnRSV6sR1iaBs_16slbWJX3WC6RmC76x8MlHn1fLBK_lU_IeZChZqLCwqUC3CXbk17OkTkdeCqFyZSS5rlfm-EiPvam6iW51tA-ywxrmUjkCDE_k-zZIZTVVSg.3YAfgcQ4ZkPtYPNvOzWCLQ
+      required: true
+      description: >-
+        an JWT which has been signed with the clients signing private key and encrypted with the F2F public
+        encryption key.
+      schema:
+        type: string
+    Authorization:
+      in: header
+      name: Authorization
+      example: bearer abcdefghijklmnopqrstuvwxyz
+      required: true
+      description: >-
+        OAuth Bearer token - this it the `authorizationCode` returned by `POST /claimedIdentity`
```

Comment:
no ref to claimedidentity

Reply:
This is present in the OpenAPI spec for F2F, see [here](https://github.com/alphagov/digital-identity-architecture/pull/263/files#diff-3d2c24e45da48c1c960ba277a64a09751e485713c51c0c25992bda5cb7ef67c5R1151)